### DISCUSSION TITLE
Graphql API schema fetching - 500 internal server error

### DISCUSSION THREAD

**Discussion by leartbeqiraj1** [community/community#78259] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

Graphql [Explorer](https://docs.github.com/en/graphql/overview/explorer) is throwing 'Error fetching schema' error. While I can execute queries, I can't fetch schema. I don't have access to live syntax, nor highlighted validation errors either." In the status page, API seems to be healthy, but it seems like it's actually not (or is it just me).

Request to fetch fails with status: '500 Internal Server Error':
![image](https://github.com/community/community/assets/126204087/38ff0e98-838e-4822-9621-bfc606340f99)

**Comment by AlmedinShala** [comment]
Having the same issue!

**Comment by leartbeqiraj1** [comment]
It looks like it fixed itself, closing this discussion
